docs/topics/whatsnew/whatsnew22.md

```diff
@@ -235,7 +235,7 @@ This feature tells the compiler to apply the annotation to all relevant parts of
 
 * **`get`**: the getter method.
 
-* **`set_param`**: the parameter of the setter method, if the property is defined as `var`.
+* **`setparam`**: the parameter of the setter method, if the property is defined as `var`.
```

Comment:
IMO
I think `setparam` is the correct use-site target name

Reply:
Yes, it is! We fixed the KEEP as well. https://github.com/Kotlin/KEEP/commit/568ecac18c771f4805f8d1de47738d108f170126

Reply:
I merged your changes from `master` branch